contracts/utils/Strings.sol

```diff
@@ -37,6 +38,13 @@ library Strings {
         }
     }
 
+    /**
+     * @dev Converts a `int256` to its ASCII `string` decimal representation.
+     */
+    function toString(int256 value) internal pure returns (string memory) {
+        return string(abi.encodePacked(value < 0 ? "-" : "",toString(SignedMath.abs(value))));       
```

Comment:
any reason to do `string(abi.encodePacked(...))` and not `string.concat(...)`. I think the second one is clearer and cleaner

Reply:
I used `abi.encodePacked` because it works with previous version of solidity and if a beginner wants to use the library, they aren’t faced with much difficulty. But on second thought, it's better a beginner gets used to the new implementation. 

I have made a new commit

Reply:
That would require bumping the pragma from `pragma solidity ^0.8.0;` to `pragma solidity ^0.8.12;` ...

It not just a matter of beginner devs, but also of good support with big projects that are fixing a solidity version that is before 0.8.12.

Considering we can do without the pragma bump, you might be right that we should probable use `string(abi.encodePacked(...))`.